it/controllers/agent/incomeSources/add/AddBusinessNameControllerISpec.scala

```diff
@@ -119,8 +119,9 @@ class AddBusinessNameControllerISpec extends ComponentSpecBase {
       }
 
       val result = IncomeTaxViewChangeFrontend.post("/income-sources/add/business-name", clientDetailsWithConfirmation)(formData)
+
```

Comment:
Can you add a check for business name ? check with database using the session service